examples/tv-casting-app/android/App/app/src/main/java/com/matter/casting/ConnectionExampleFragment.java

```diff
@@ -56,6 +58,11 @@ public class ConnectionExampleFragment extends Fragment {
   private TextView connectionFragmentStatusTextView;
   private Button connectionFragmentNextButton;
 
+  // Get a singleton instance of the MatterCastingPlayerDiscovery. Which can be used to call
+  // stopDiscovery() during connection.
+  static final CastingPlayerDiscovery matterCastingPlayerDiscovery =
+      MatterCastingPlayerDiscovery.getInstance();
```

Comment:
It is more idiomatic to directly call MatterCastingPlayerDiscovery.getInstance() inline where you want to use it given the singleton pattern employed.

Reply:
removing the stop discovery button form the Connection page. This is no longer needed.